selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/details/WorkspaceDetailsOverviewTest.java

```diff
@@ -115,40 +65,20 @@ public void shouldCreateWorkspaceAndOpenOverviewPage() {
     newWorkspace.waitPageLoad();
     newWorkspace.typeWorkspaceName(WORKSPACE_NAME);
 
-    selectDevfileAndCheckWorkspaceName(Devfile.APACHE_CAMEL);
+    selectDevfileAndCheckWorkspaceName(Devfile.JAVA_MAVEN);
 
     selectDevfileAndCheckWorkspaceName(Devfile.JAVA_GRADLE);
 
     // create workspace
-    addOrImportForm.clickOnAddOrImportProjectButton();
-    addOrImportForm.addSampleToWorkspace(SAMPLE_NAME);
     newWorkspace.clickOnCreateButtonAndEditWorkspace();
     workspaceOverview.checkNameWorkspace(WORKSPACE_NAME);
   }
 
-  @Test(priority = 2)
```

Comment:
Do we have another test to check wrong workspace name handling?

Reply:
Yes. We have `RenameWorkspaceTest` selenium test to check workspace renaming